rust/operator-binary/src/security/kerberos.rs

```diff
@@ -88,12 +88,6 @@ impl CoreSiteConfigBuilder {
                 .add(
                     "dfs.namenode.kerberos.principal.pattern",
                     format!("nn/{principal_host_part}"),
-                )
-                // Otherwise we fail with `java.io.IOException: No groups found for user nn`
-                // Default value is `dr.who=`, so we include that here
```

Comment:
I think we need to add a comment here to make this clear: are we assuming that any static mapping that **is** required, is the responsibility of the person writing the custom resource? Or is this not required anymore with the opa authorizer?

Reply:
I *think* it is not required any more, so I don't expect the user to do anything at all

Reply:
Changed as discussed in bb264da

Reply:
Thanks!